content/en/docs/contribute/generate-ref-docs/quickstart.md

```diff
@@ -244,6 +248,75 @@ A few minutes after your pull request is merged, your updated reference
 topics will be visible in the
 [published documentation](/docs/home/).
 
+## Method 2: Working in a containerized environment
+
+There are a series of steps available if you wish to save some time
+by not installing the dependencies
+
+Please follow the initial steps of cloning the
+documentation repository from the local environment build tutorial.
+Considering you have docker installed and running, We can proceed on to building the container image
+
+### Change directories to update-imported-docs
+
+Make sure you are inside your documentation repository, Then
+
+``` shell
+cd ./update-imported-docs
+```
+
+Here you shall find all the build tools including the scripts and the dockerfile
+
+### Build an image from the dockerfile
+
+You can start building an image for the release you wish to generate
+
+```shell
+docker build -t refdocs:<release-version> --build-arg release=<release-version> . 
+```
+
+**Note**: Replace the 'release-version' tags with the release version you wish to build, 
+Eg: ```docker build -t refdocs:1.28 --build-arg release=1.28 .```
+
+You'll see your image in your docker dashboard after a successful build
+
+### Running the image
+
+What remains is running the image and generating your documentation, 
+You can execute the shell command below to get your image up and running
+
+**Note**: Since the following command uses a ```$(pwd)``` reference,
+Please ensure you are inside the update-imported-docs directory while executing it.
+_Alternatively_, You can also provide the absolute path to your documentation repository
+
+```shell
+docker run --name=refdocsgenerator -v "${pwd}/../:/website" -v sharedVolume:/var/lib/docker/volumes/sharedVolume/_data  -v /var/run/docker.sock:/var/run/docker.sock refdocs:<release-version>
```

Comment:
Well I agree @sftim, The change does seem controversial at first because it involves mounting a socket
But considering this is not a part of any heavy production and just generates some documentation, We _can_ let this pass imo

That being said, I'm open to any and all suggestion. If needed, We can change our approach and I'll figure something out

Reply:
Mounting the Docker socket into a container is (can be) an elevation of privilege risk. See https://blog.quarkslab.com/why-is-exposing-the-docker-socket-a-really-bad-idea.html (there are many similar articles).

Reply:
No worries @sftim, Let's change our approach a bit. I'll figure something out